cloud/google/util.sh

```diff
@@ -10,18 +10,21 @@ function Header() {
 function WaitForStatus() {
   TYPE=$1
   NAME=$2
-  WANT_STATUS=$3
+  ZONE=$3
+  WANT_STATUS=$4
   STATUS=""
   until [[ "${STATUS}" =~ "${WANT_STATUS}" ]]; do
     sleep 1
-    STATUS=$(${GCLOUD} compute ${TYPE} describe ${NAME} | grep "status:")
+    STATUS=$(${GCLOUD} compute ${TYPE} describe ${NAME} --zone ${ZONE} | grep "status:")
```

Comment:
I think all greps should do what I'm after here (or it has no business calling itself grep!)
Plus if I wanted the script to use my own 1337 grep I'd probably try shoving it in the `PATH` first.